components/atom/backToTop/src/index.js

```diff
@@ -18,11 +18,12 @@ const STYLES = {
 
 const AtomBackToTop = ({
   iconTop: IconTop,
+  minHeight,
   textTop,
-  scrollSteps,
-  scrollIntervalTime,
-  refContainer,
-  style
+  scrollSteps = 100,
+  scrollIntervalTime = 50,
+  refContainer = document.body,
```

Comment:
I realized now that this could potentially be broken on client-side rendering.

Reply:
So is it better to define default values on `.defaultProps`?

Reply:
Or the problem is `document.body`